samples/Sentry.Samples.Maui/Sentry.Samples.Maui.csproj

```diff
@@ -67,7 +67,7 @@
     <RuntimeIdentifier Condition="'$(TargetPlatformIdentifier)' == 'ios'          And '$(OSArchitecture)' == 'Arm64' And '$(_IsPublishing)' != 'true'">iossimulator-arm64</RuntimeIdentifier>
     <RuntimeIdentifier Condition="'$(TargetPlatformIdentifier)' == 'maccatalyst'  And '$(OSArchitecture)' == 'Arm64'">maccatalyst-arm64</RuntimeIdentifier>
 
-    <RuntimeIdentifier Condition="'$(TargetPlatformIdentifier)' == 'android'      And '$(OSArchitecture)' == 'x64'">android-x64</RuntimeIdentifier>
+    <RuntimeIdentifiers Condition="'$(TargetPlatformIdentifier)' == 'android'     And '$(OSArchitecture)' == 'x64'">android-arm;android-arm64;android-x86;android-x64</RuntimeIdentifiers>
```

Comment:
@bricefriha why can we not do this like all the other runtimeidentirier elements here? So why can't we have a separate line for x64 and x86? And why is the `android-arm` identifier required?

Reply:
>  So why can't we have a separate line for x64 and x86?

It seems to have been a mistake. thanks for pointing this out

> why is the android-arm identifier required?

To be fair, we could remove it; it's more for backward compatibility. I don't think there are many people who would debug using a 32-bit device, so we could remove this

Reply:
I see, yeah let's just add what we need to keep it as simple as possible. We can add it if/when it's ever needed in the future.

Reply:
Sounds good